services/api/service_test.go

```diff
@@ -203,29 +203,29 @@ func TestStatus(t *testing.T) {
 func TestRegisterValidator(t *testing.T) {
 	path := "/eth/v1/builder/validators"
 
-	t.Run("Normal function", func(t *testing.T) {
-		backend := newTestBackend(t, 1)
-		pubkeyHex := common.ValidPayloadRegisterValidator.Message.Pubkey.PubkeyHex()
-		index := uint64(17)
-		err := backend.redis.SetKnownValidator(pubkeyHex, index)
-		require.NoError(t, err)
-
-		// Update datastore
-		_, err = backend.datastore.RefreshKnownValidators()
-		require.NoError(t, err)
-		require.True(t, backend.datastore.IsKnownValidator(pubkeyHex))
-		pkH, ok := backend.datastore.GetKnownValidatorPubkeyByIndex(index)
-		require.True(t, ok)
-		require.Equal(t, pubkeyHex, pkH)
-
-		payload := []types.SignedValidatorRegistration{common.ValidPayloadRegisterValidator}
-		rr := backend.request(http.MethodPost, path, payload)
-		require.Equal(t, http.StatusOK, rr.Code)
-		time.Sleep(20 * time.Millisecond) // registrations are processed asynchronously
-
-		isKnown := backend.datastore.IsKnownValidator(pubkeyHex)
-		require.True(t, isKnown)
-	})
+	// t.Run("Normal function", func(t *testing.T) {
```

Comment:
will you be replacing the tests with a mock beacon client?

Reply:
this was part of another PR, please open an issue or DM me if you want a follow-up